README.md

```diff
@@ -137,6 +137,7 @@ format specifications to be used.
 - [Synalyze It!](https://www.synalysis.net/) - Hex editor with templates for binary analysis
 - [wxHex Editor](https://www.wxhexeditor.org/) - Cross-platform editor with file comparison
 - [hexalepis](https://github.com/sealmove/hexalepis) - Win/Unix gui+terminal, [tweak](https://www.chiark.greenend.org.uk/~sgtatham/tweak/btree.html) engine, .ksy visualization
+- [Hex Editor Neo](https://www.hhdsoftware.com/hex-editor) - fast binary file editor for Windows (supports Kaitai Struct in its [Structure Viewer](https://docs.hhdsoftware.com/hex/definitive-guide/structure-viewer/kaitai.html))
```

Comment:
Minor, but I'll suggest it anyway (I personally distinguish these things, but it may be nonsense to declare my approach as the only "correct") - if the link only contains the object, I expect to land on a general page describing the object. However, if it has a strong connection to what have been written before, I include the preposition to suggest that:

```suggestion
- [Hex Editor Neo](https://www.hhdsoftware.com/hex-editor) - fast binary file editor for Windows (supports Kaitai Struct [in its Structure Viewer](https://docs.hhdsoftware.com/hex/definitive-guide/structure-viewer/kaitai.html))
```